lib/evaluation/abstractEvaluationHandler.ts

```diff
@@ -28,9 +28,13 @@ export abstract class AbstractEvaluationHandler implements EvaluationHandler {
     return this._client;
   }
 
+  public set client(client: EvaluationClient) {
```

Comment:
why did you add this method? we populating it in the constructor

Reply:
No. Probably a rebase problem.